### ISSUE TITLE
To-do list stays persistent

### ISSUE THREAD

**Issue by LividJava** [MadeBaruna/paimon-moe#187]
The To-do list persists after expanding resin requirements and going to settings.
See video for reference, not sure what's causing this it could just be on my end given my plugins.

https://user-images.githubusercontent.com/7604544/142744459-0d61563d-cb2a-483c-9652-b200ddc61497.mp4

**Comment by LividJava** [comment]
Apologies for the poor video, It was just quick and dirty to demonstrate the issue

**Comment by npanuhin** [comment]
Hey! Do you still have this problem? If so, can I ask you to try clearing cookies and cache in your browser

**Comment by LividJava** [comment]
Yes this issue is still present after clearing cookies and cache.

**Comment by LividJava** [comment]
Updated the issue with steps to reproduce, this is present in chromium based browsers as well.
